analyzer/src/main/kotlin/managers/Npm.kt

```diff
@@ -163,6 +163,23 @@ open class Npm(
         }
     }
 
+    private fun parseLicenses(json: JsonNode): SortedSet<String> {
+        val declaredLicenses = sortedSetOf<String>()
+
+        // See https://docs.npmjs.com/files/package.json#license.
+        json["license"]?.let { licenseNode ->
+            val type = licenseNode.textValue() ?: licenseNode["type"].textValueOrEmpty()
+            declaredLicenses += type
```

Comment:
This adds an empty string to `declaredLicenses` if `licenseNode.textValue()` and `licenseNode["type"]` are `null`. Would be a good opportunity to fix this issue:

```kotlin
(licenseNode.textValue() ?: licenseNode["type"].textValue())?.let { declaredLicenses += it }
```

Reply:
Good finding, fixed in a new commit.